crates/bevy_text/src/text.rs

```diff
@@ -324,11 +329,39 @@ impl Default for TextFont {
         Self {
             font: Default::default(),
             font_size: 20.0,
+            line_height: LineHeight::default(),
             font_smoothing: Default::default(),
         }
     }
 }
 
+/// Specifies the height of each line of text for `Text` and `Text2d`
+///
+/// Default is 1.2x the font size
+#[derive(Debug, Clone, Copy, Reflect)]
+#[reflect(Debug)]
+pub enum LineHeight {
+    /// Set line height to a specific number of pixels
+    Px(f32),
+    /// Set line height to a multiple of the font size
+    Scalar(f32),
```

Comment:
I am not sure that the name `Scalar` makes sense here.

Maybe `Relative`? `Em`? I will ask other UI people to weigh in.

Reply:
FontRelative gets my vote.

Reply:
`Relative` / `Em` might be a bit confusing and `FontRelative` seems a bit odd.

`RelativeToFontSIze` ? It's a bit long but completely unambiguous.

Reply:
In CSS, the equivalent here seems to be just a unitless number. So this at least shouldn't be called `Em`, since that behaves differently.
<https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#values>

I think `Relative`, `RelativeToFont`, or `Factor` would be my suggestion

Reply:
Yeah `RelativeToFont` seems fine to me.